plugins/callback/opentelemetry.py

```diff
@@ -330,7 +330,7 @@ def update_span_data(self, task_data, host_data, span, disable_logs):
         self.add_attributes_for_service_map_if_possible(span, task_data)
         # Send logs
         if not disable_logs:
-            span.add_event(task_data.dump)
+            span.add_event(task_data.dump, attributes={"ansible.task.name": name, "ansible.task.host.name": host_data.name})
```

Comment:
`ansible.task.host.name` might not be needed since it's part of the `name` variable defined in https://github.com/ansible-collections/community.general/pull/6531/files#diff-82d3d091bae7417ded982278ccf8f928f23b64a2ce6a967be1bd84ade2266e70R288

> name = '[%s] %s: %s' % (host_data.name, task_data.play, task_data.name)